templates/unpackerr.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0"?>
+<Container version="2">
+  <Name>unpackerr</Name>
+  <Repository>golift/unpackerr</Repository>
+  <Registry>https://hub.docker.com/r/golift/unpackerr</Registry>
+  <Network>bridge</Network>
+  <Shell>sh</Shell>
+  <Privileged>false</Privileged>
+  <Support>https://github.com/davidnewhall/unpackerr/issues/new</Support>
+  <Project>https://github.com/davidnewhall/unpackerr</Project>
+  <Icon>https://raw.githubusercontent.com/wiki/davidnewhall/unpackerr/images/logo.png</Icon>
+  <Category>Tools:</Category>
+  <Description>Download extractor for Sonarr, Radarr and Lidarr!</Description>
+  <Overview>
+  RAR download extractor for Sonarr, Radarr and Lidarr! Mount /downloads on all 3 or 4 containers to the same path on your host.
+  This application will automatically poll all of the apps (Lidarr, Sonarr, Radarr) to check for downloads. When a completed
+  download contains a rar file, this application will extract the file and move the extracted contents back into the download
+  location. Files will be extracted recursively in an attempt to get all subtitles. [b]If you don't use one of Lidarr, Sonarr
+  or Radarr, simply empty those two fields for that app and it wont be polled.[/b] If you put all your containers on the same
+  custom bridge you may not need to expose any ports, and you can use DNS to reach other containers. It's a very convenient configuration!
+  </Overview>
+  <Config Name="Downloads Location" Target="/downloads" Default="" Mode="rw" Description="Container Path: /downloads - this must be the same path on this app and sonarr/radarr/lidarr!" Type="Path" Display="always-hide" Required="true" Mask="false"></Config>
```

Comment:
I'd suggest to change from Display=alway-hide to Display=always. The container path is not always `/downloads` and it would be nice to be able to edit the path without switching to advanced view